sdk/src/beta9/cli/config.py

```diff
@@ -175,3 +179,38 @@ def select_context(name: str, config_path: Path):
     save_config(contexts=contexts, path=config_path)
 
     terminal.success(f"Default context updated with '{name}'.")
+
+
+@management.command(
+    name="export-config",
+    help="Export the current workspace configuration",
+)
+@click.option(
+    "--name",
+    type=click.STRING,
+    help="Name of the config cntext to export. Default is the `default` context.",
+    default=DEFAULT_CONTEXT_NAME,
+    required=False,
+)
+@extraclick.pass_service_client
+def export_config(
+    service: ServiceClient,
+    name: str,
+):
+    contexts = load_config()
+    context = contexts.get(name)
+
+    if not context:
+        terminal.error(f"Context '{name}' does not exist.")
+
+    resp = service.gateway.export_workspace_config(ExportWorkspaceConfigRequest())
+    config = {
```

Comment:
Can you look at using `resp.to_json(indent=2, casing=Casing.SNAKE)` instead of the json.dumps and print at the end?

Reply:
i need to add in the `token` variable before dumpnig it

Reply:
Oh right, so maybe to_dict, then json.dump it?

Reply:
done